libs/full/runtime_configuration/src/runtime_configuration.cpp

```diff
@@ -380,12 +380,16 @@ namespace hpx { namespace util {
         // clang-format off
 #if defined(HPX_HAVE_LOGGING)
 #define HPX_TIMEFORMAT "$hh:$mm.$ss.$mili"
+#if defined(HPX_HAVE_DISTRIBUTED_RUNTIME)
```

Comment:
I was hoping we could get rid of this definition after the recent changes (and #5313). Can we print  a reasonable default for the local-only case?

Reply:
I'm fine with getting rid of  `HPX_HAVE_DISTRIBUTED_RUNTIME`. This code here is used by both, the local and distributed runtime, however. Are you suggesting to always print `--------` or similar even in the local case?

Reply:
Yes, something like that. I think currently there is just nothing printed in the place of `hpxcomponent` with the distributed runtime off, which is also fine by me.

Reply:
@msimberg Could you please have another look? I have removed the `#if` and I hope that it is now as you envisioned it. Please let me know.